### PR TITLE
Html assignment

### PR DESCRIPTION
**Pull request by Naveen-Goud**
html and css assignment completed

### CODE REVIEW

File: htmlassignment/contactme.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            73 73 637 637
+            91 – 8413 – 235300
+            91 – 8413 -235399
+            Extension Numbers
+            <br><br>
+            Email
+            info@vjit.ac.in
+        </div>
+      
+      <!-- <div class="formBox">
```
Comment (msaisrinivas) [inline htmlassignment/contactme.html]: Remove the commented lines if you don't use them

File: htmlassignment/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <ul>
+        <li><a class="active" href="../htmlassignment/index.html">Home</a></li>
+        <li><a   href="../htmlassignment/aboutme.html">About Me</a></li>
+        <li><a href="../htmlassignment/contactme.html">Contact</a></li>
+        <li><a href="../htmlassignment/gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+    <section><br><br>
+      <div  class="table">
+          <!-- <tr><td>Research</td></tr> -->
```
Comment (msaisrinivas) [inline htmlassignment/index.html:22]: Remove the commented lines

File: htmlassignment/gallery.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </nav>
+    <h1><center>PHOTOGRAPHS OF  VJIT</center></h1>
+    <div class="gal">
+       
+           <div style="flex-shrink: 5"><img src="../images/cblock.jpeg"></div>
+           <div style="flex-shrink: 5"><img src="../images/vjitlogo.jpeg"></div>
+            <div style="flex-shrink: 10"><img src="../images/auditorium.jpg"></div>
+            <div style="flex-shrink: 2"> <img src="../images/library.jpg"></div>
+          
+            <div><img src="../images/events.jpg"></div>
```
Comment (msaisrinivas) [inline htmlassignment/gallery.html]: format the HTML files (using prettier)

**Review by msaisrinivas — CHANGES_REQUESTED** [review]

File: htmlassignment/contactme.html (hunk 1 above)
Comment (Naveen-Goud) [reply htmlassignment/contactme.html]: commented lines are removed

File: htmlassignment/index.html (hunk 2 above)
Comment (Naveen-Goud) [reply htmlassignment/index.html:22]: commented lines are removed.

**Review by msaisrinivas — APPROVED** [review]

**Pull request merged**